be/src/olap/compaction.cpp

```diff
@@ -625,6 +625,10 @@ Status Compaction::modify_rowsets(const Merger::Statistics* stats) {
 
     {
         std::shared_lock rlock(_tablet->get_header_lock());
+        if (compaction_type() == ReaderType::READER_CUMULATIVE_COMPACTION &&
+            config::compaction_policy == CUMULATIVE_TIME_SERIES_POLICY) {
+            _tablet->save_cumulative_layer_point(_output_rowset->end_version() + 1);
```

Comment:
why add this method? because in line 632, it will call save_meta method, I think you could just modify the memory state and it will be saved in save_meta?